src/lib/prompts.js

```diff
@@ -42,6 +43,23 @@ const prompts = {
           .trim()
           .replace(/\s{1,}/g, '-'),
     },
+    {
+      type: 'select',
+      name: 'networkId',
+      initial: 0, // 0 = testnet, 1 = mainnet, change it to 1 after the HF
+      choices: Constants.networkIds.map((networkId) => ({
+        name: capitalize(networkId),
+        value: networkId,
+      })),
+      message: (state) => {
+        const style =
+          state.submitted && !state.cancelled ? chalk.green : chalk.reset;
+        return style('Choose the target network ID:');
```

Comment:
```suggestion
        return style('Choose the target network:');
```

Reply:
I suggest we just ask the user to select a target network. Network id is more of an internal term we use in the code.